README.md

```diff
@@ -9,6 +9,26 @@ This repository provides example notebooks and "getting started" code
 samples to help you integrate RAPIDS with the hyperparameter
 optimization services from Azure ML, AWS Sagemaker, Google
 Cloud, and Databricks. The directory for each cloud contains a step-by-step guide to
+launch an example hyperparameter optimization job.
+
+Each example job will use RAPIDS
+[cuDF](https://github.com/rapidsai/cudf) to load and preprocess 20
+million rows of airline arrival and departure data and build a model
+to predict whether or not a flight will arrive on time (dataset is sourced from [Airline On-Time Performance Data from BTS](https://transtats.bts.gov/Tables.asp?DB_ID=120&DB_Name=Airline%20On-Time%20Performance%20Data&DB_Short_Name=On-Time#)). It
+demonstrates both [cuML](https://github.com/rapidsai/cuml) Random
+Forests and GPU-accelerated XGBoost modeling.
+
+## Microsoft Azure ML
```

Comment:
I think all these ## <TOPIC> blocks are out of data as well, and can probably be removed. Looks like some extra stuff sneaked in with the merge.